packages/chip/src/Chip/Chip.styles.ts

```diff
@@ -39,175 +46,270 @@ export const chipWrapperPadding: Record<
+      },
+      [Property.Text]: {
+        [InteractionState.Default]: palette.green.light2,
+      },
+      [Property.Icon]: {
+        [InteractionState.Default]: palette.green.light2,
+      },
     },
     [Theme.Light]: {
-      color: palette.green.dark3,
-      bgColor: palette.green.light3,
-      focusBgColor: palette.green.light1,
+      [Property.Background]: {
+        [InteractionState.Default]: palette.green.light3,
+        [InteractionState.Focus]: palette.green.light1,
+      },
+      [Property.Text]: {
+        [InteractionState.Default]: palette.green.dark3,
+      },
+      [Property.Icon]: {
+        [InteractionState.Default]: palette.green.dark2,
+      },
     },
   },
   [Variant.Purple]: {
     [Theme.Dark]: {
-      color: palette.purple.light2,
-      bgColor: palette.purple.dark3,
-      focusBgColor: '#892CCA',
+      [Property.Background]: {
+        [InteractionState.Default]: palette.purple.dark3,
+        [InteractionState.Focus]: '#892CCA',
+      },
+      [Property.Text]: {
+        [InteractionState.Default]: palette.purple.light2,
+      },
+      [Property.Icon]: {
+        [InteractionState.Default]: palette.purple.light2,
+      },
     },
     [Theme.Light]: {
-      color: palette.purple.dark3,
-      bgColor: palette.purple.light3,
-      focusBgColor: '#E19AFF',
+      [Property.Background]: {
+        [InteractionState.Default]: palette.purple.light3,
+        [InteractionState.Focus]: '#E19AFF',
+      },
+      [Property.Text]: {
+        [InteractionState.Default]: palette.purple.dark3,
+      },
+      [Property.Icon]: {
+        [InteractionState.Default]: palette.purple.dark2,
+      },
     },
   },
   [Variant.Yellow]: {
     [Theme.Dark]: {
-      color: palette.yellow.light2,
-      bgColor: palette.yellow.dark3,
-      focusBgColor: '#C27823',
+      [Property.Background]: {
+        [InteractionState.Default]: palette.yellow.dark3,
+        [InteractionState.Focus]: '#C27823',
+      },
+      [Property.Text]: {
+        [InteractionState.Default]: palette.yellow.light2,
+      },
+      [Property.Icon]: {
+        [InteractionState.Default]: palette.yellow.light2,
+      },
     },
     [Theme.Light]: {
-      color: palette.yellow.dark3,
-      bgColor: palette.yellow.light3,
-      focusBgColor: '#FFD664',
+      [Property.Background]: {
+        [InteractionState.Default]: palette.yellow.light3,
+        [InteractionState.Focus]: '#FFD664',
+      },
+      [Property.Text]: {
+        [InteractionState.Default]: palette.yellow.dark3,
+      },
+      [Property.Icon]: {
+        [InteractionState.Default]: palette.yellow.dark2,
+      },
     },
   },
+} as const satisfies RecursiveRecord<
+  [Variant, Theme, Property, InteractionState, string],
+  false
+>;
+
+const truncateDisabledColor = {
+  [Theme.Dark]: palette.gray.dark1,
+  [Theme.Light]: palette.gray.light1,
+};
+
+const wrapperDisabledColor = {
+  [Theme.Dark]: palette.gray.dark2,
+  [Theme.Light]: palette.gray.base,
 };
```

Comment:
these look like they should use tokens

Reply:
I'll double-check with rob because they currently don't match any of our tokens